timelock-agent/src/main/java/com/palantir/timelock/config/TimeLockInstallConfiguration.java

```diff
@@ -35,7 +35,11 @@
 
     PaxosInstallConfiguration paxos();
 
-    ClusterConfiguration cluster();
+    @Deprecated
```

Comment:
Normally we tell people what to do instead when marking something as deprecated.